frontend/src/app/models/corpus.ts

```diff
@@ -146,5 +146,20 @@ export interface CorpusDocumentationPage {
     corpus: string;
     type: string;
     content: string;
+    content_template: string,
     index?: number;
 }
+
+export interface CorpusDocumentationPageEditable {
+    id?: number,
+    type: string,
+    corpus: string,
+    content_template: string,
+};
+
+
+export interface CorpusDocumentationPageSubmitData {
+    type: string,
+    corpus: string,
+    content_template: string,
+};
```

Comment:
Nitpicky, but the first interface oculd extend the second to avoid some repetition.